hardware/raspberrypi/spi/README.md

```diff
@@ -155,26 +187,12 @@ Bidirectional or "3-wire" mode is supported by the spi-bcm2835 kernel module. Pl
 
 ### Transfer modes
 
-Only interrupt mode is supported.
-
-### Deprecated warning
-
-The following appears in the kernel log:
-
-```
-bcm2708_spi bcm2708_spi.0: master is unqueued, this is deprecated
-```
+Interrupt mode is supported on all SPI buses. SPI0 also supports DMA transfers.
 
 ### SPI driver latency
 
 This [thread](https://www.raspberrypi.org/forums/viewtopic.php?f=44&t=19489) discusses latency problems.
 
```

Comment:
Probably need to move data in to this doc from the thread.

Reply:
Feel free.